config/opal_mca.m4

```diff
@@ -176,14 +179,16 @@ of type-component pairs.  For example, --enable-mca-no-build=pml-ob1])
         msg="none (dlopen disabled)"
     elif test -z "$enable_mca_dso"; then
         DSO_all=0
-        msg=default
+        DSO_accelerator=1
```

Comment:
Should this be:
```suggestion
enable_mca_dso=accelerator
```
?

Reply:
that makes things a little complicated since we'd need to repeat the logic at 
```
for item in $enable_mca_dso; do
            AS_VAR_SET([AS_TR_SH([DSO_$item])], [1])
            msg="$item $msg"
        done
        ```
a second time or move that out of the if block and put in a separate one below.